src/m365/aad/commands/user/user-set.spec.ts

```diff
@@ -291,4 +388,70 @@ describe(commands.USER_SET, () => {
     } as any);
     assert(loggerLogSpy.notCalled);
   });
+
+  it('updates Azure AD user and set its manager by id', async () => {
+    sinon.stub(request, 'patch').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/users/${formatting.encodeQueryParameter(userPrincipalName)}`) {
+        return userResponse;
+      }
+
+      throw 'Invalid request';
+    });
+
+    const putSpy = sinon.stub(request, 'put').callsFake(async (opts) => {
+      if (opts.url === `https://graph.microsoft.com/v1.0/users/${userPrincipalName}/manager/$ref`) {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, { options: { userPrincipalName: userPrincipalName, displayName: displayName, managerUserId: managerUserId } });
```

Comment:
Let's try to test one thing at a time. So in this case, let's remove displayName.